apps/fm/test/unit/fm_test.js

```diff
@@ -338,6 +352,20 @@ suite('FM', function() {
       test('#airplane-mode-warning is hidden', function() {
         assert.equal(!!$('airplane-mode-warning').hidden, true);
       });
+
+      test('#antenna-warning is on and container is hidden', function() {
```

Comment:
Depending upon the response to the above comment, (i.e. if we decide not to show both warnings at the same time), aren't both permutations taken care of in the following sense...?

- If airplane mode is on, as it's the top-most layer in the z-index, we only really need to know that both the container and the antenna-warning are hidden.  We're not really concerned if the antenna-warning is on because it'll never stack higher than the airplane-mode-warning

- If airplane mode is off, now we need to check if the antenna-warning is on, so as to hide itself and the container appropriately when it's on/off.